spec/factories/notes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :note do
+    user
+    title { Faker::Book.title }
+    content { Faker::Lorem.sentence }
+    note_type { :review }
```

Comment:
`note_type` es de tipo `enum`, podes acceder a los valores del mismo si le das una miradita a esa doc:
https://blog.saeloun.com/2022/01/05/how-to-use-enums-in-rails/

Y combinando con estos métodos, podrías crear notas con los tipos de forma aleatoria:
https://apidock.com/ruby/Hash/values y https://apidock.com/ruby/Array/sample